app/(request)/public-cloud/edit/[licencePlate]/roles/Admins/page.tsx

```diff
@@ -0,0 +1,28 @@
+import TableAWSRoles from '@/components/table/TableAWSRoles';
+import {
+  getSubGroupMembersByLicencePlateAndName,
+  getPublicCloudProjectUsers,
+  User,
+} from '@/app/api/public-cloud/aws-roles/routes';
+import TableBodyAWSRoles from '@/components/table/TableBodyAWSRoles';
+export default async function ProductAWSRoles() {
```

Comment:
I believe this page is only for the `admin` user?
Let's add the session & permission checks.